src/views/partials/header.hbs

```diff
@@ -0,0 +1,71 @@
+<header>
```

Comment:
We need to change our mindset from a code organisation perspective.
I appreciate that you have tried to organise the code into multiple separate files. 

According to me, it actually is not a true separation of concern but rather separation of stack / technology.

Eg:
the only job of cart.css, cart.js and cart.hbs  is to work together and render cart. Then won't it be better if all three of them reside in the same folder ?